@here/harp-mapview/lib/MapView.ts

```diff
@@ -1507,9 +1513,27 @@ export class MapView extends THREE.EventDispatcher {
 
     /**
      * Get distance from camera to the point of focus in world units.
+     *
+     * @note If camera does not point to any ground anymore the last focus point distance is
+     * then returned.
+     *
+     * @returns Last known focus point distance.
+     */
+    get targetDistance(): number {
+        return this.m_targetDistance;
+    }
+
+    /**
+     * Get world coordinates of camera focus point.
+     *
+     * @note The focus point coordinates are updated with each camera update so you don't need
+     * to re-calculate it, although if the camera started looking to the void, the last focus
+     * point is stored.
+     *
+     * @returns world coordinates of last camera focus point.
      */
-    get lookAtDistance(): number {
-        return this.m_lookAtDistance;
+    get targetPoint(): THREE.Vector3 {
```

Comment:
Maybe call it targetWorld to make more clear that it is in world space

Reply:
Please also add a getter for the target in geo space as agreed here:
https://confluence.in.here.com/display/HARP/Consistent+naming+of+camera+properties

Reply:
There is also big overlap with this PR: https://github.com/heremaps/harp.gl/pull/1023

Reply:
I also created a draft PR that might help to continue the discussion: https://github.com/heremaps/harp.gl/pull/1178

Reply:
Ok, I have provided the already used convention for getters:
- geoCenter / worldCenter
- geoTarget / worldTarget.